pytest/tests/loadtest/locust/common/ft.py

```diff
@@ -1,18 +1,67 @@
 import json
+import random
 import sys
 import pathlib
+from locust import events
 
 sys.path.append(str(pathlib.Path(__file__).resolve().parents[4] / 'lib'))
 
+import cluster
+import key
 import transaction
 
 from account import TGAS
-from common.base import Transaction
+from common.base import Account, CreateSubAccount, Deploy, NearUser, send_transaction, Transaction
+
+
+class FTContract:
+    # NEAR balance given to contracts, doesn't have to be much since users are
+    # going to pay for storage
+    INIT_BALANCE = NearUser.INIT_BALANCE
+
+    def __init__(self, account: Account, code):
+        self.account = account
+        self.registered_users = []
+        self.code = code
+
+    def install(self, node):
+        """
+        Deploy and initialize the contract on chain.
+        The account should already exist at this point.
+        """
+        self.account.refresh_nonce(node)
+        send_transaction(node, Deploy(self.account, self.code, "FT"))
+        send_transaction(node, InitFT(self.account))
+
+    def register_user(self, user: NearUser):
+        user.send_tx(InitFTAccount(self.account, user.account),
+                     locust_name="Init FT Account")
+        user_account_id = user.account_id
```

Comment:
I think this variable doesn't really help with readability, so unless we're following [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter), I think we can remove it :)